app/views/finders/show.html.erb

```diff
@@ -62,6 +62,7 @@
 </header>
 
 <form method="get" action="<%= finder.slug %>" class="js-live-search-form">
+  <div class="hidden"><input type="text" name="parent" value="<%= @parent %>"></div>
```

Comment:
Instead of wrapping this in a hidden div, could this just be `<input type="hidden" ... >` ?

Reply:
Good tip, thank you @andysellick!